linux_os/guide/services/ssh/sshd_approved_macs.var

```diff
@@ -12,4 +12,5 @@ interactive: false
 
 options:
     sle12_stig: hmac-sha2-512,hmac-sha2-256
+	rhel7_stig: hmac-sha2-512,hmac-sha2-256
```

Comment:
@cyarbrough76 as the variables are the exact same, please merge the `sle12_stig` and `rhel7_stig` into the same variable called `stig`.

Reply:
@cyarbrough76 use spaces and not tabs. That's why your indentation is wrong here.

Reply:
Concur, will do.

Reply:
Did see that last message in time, will redo to address spacing

Reply:
Spacing should be good now